tests/tests/smt_tests.cpp

```diff
@@ -486,7 +535,7 @@ BOOST_AUTO_TEST_CASE( setup_apply )
 {
    try
    {
-      ACTORS( (alice)(bob) )
+      ACTORS( (alice)(bob)(xyz)(xyz2) )
```

Comment:
Is this needed?

Reply:
Yes this makes the old test pass. We now check that accounts actually exist before allowing the operation to go through. I re-wrote this test, but left the old one in there for now to ensure we do not lose code coverage.